recipes/retryhttp/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "retryhttp" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/retryhttp-{{ version }}.tar.gz
+  sha256: cf18dee1f7dcbdea25653f5f11abe6ef6a694235a75538cba4877809ee9970ce
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=21.0.0,!=24.0.0,!=34.0.0,!=34.0.1,!=34.0.2,!=34.0.3,!=34.1.0,!=34.1.1,!=34.2.0,!=34.3.0,!=34.3.1,!=34.3.2,!=36.2.0
+    - setuptools-scm >=3.4
+    - pip
+  run:
+    - python >={{ python_min }}
+    - httpx
+    - pydantic
+    - requests
+    - types-requests
+    - tenacity
+
+test:
+  imports:
+    - retryhttp
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/austind/retryhttp
+  summary: Retry potentially transient HTTP errors in Python.
```

Comment:
I'm not sure what you are trying to do but I'm a big fan of https://github.com/hynek/stamina for this kind of task.

Reply:
But thanks for the recommendation!

Reply:
> It's a dependency of `apache-airflow` (several layers of dependencies deep) so not my call, I'm afraid.

Good luck!